src/sentry/demo/data_population.py

```diff
@@ -802,6 +822,39 @@ def generate_saved_query(self, project, transaction_title, name):
             version=data["version"],
         )
 
+    def generate_saved_search(self, projects):
+        SavedSearch.objects.filter().delete()
+        global_params = saved_search_by_platform["global"]
+        for params in global_params:
+            name, query = params
+            SavedSearch.objects.get_or_create(
+                is_global=True, organization=projects[0].organization, name=name, query=query
```

Comment:
@roggenkemper Nit: could we use `self.org.id` here?

Reply:
yup!